transport-native-epoll/src/test/java/io/netty/channel/epoll/EpollSocketCloseForciblyTest.java

```diff
@@ -25,6 +25,6 @@
 public class EpollSocketCloseForciblyTest extends SocketCloseForciblyTest {
     @Override
     protected List<TestsuitePermutation.BootstrapComboFactory<ServerBootstrap, Bootstrap>> newFactories() {
-        return EpollSocketTestPermutation.INSTANCE.socket();
+        return EpollSocketTestPermutation.INSTANCE.statefulSocket();
```

Comment:
Could we just add other methods for the "new behavior" and so not need to touch these files at all ?

Reply:
Same is true for other test files

Reply:
Sure. Doing it that way will touch more files, though.